scripts/nayduck.py

```diff
@@ -205,17 +207,17 @@ def run_locally(tests):
                 '--',
                 '--exact',
                 fields[3])
-            cwd = None
+            cwd = os.path.join(REPO_DIR, 'pytest')
         elif fields[0] in ('pytest', 'mocknet'):
             fields[0] = sys.executable
-            fields[1] = os.path.join('tests', fields[1])
+            fields[1] = os.path.join('./pytest/tests', fields[1])
             cmd = fields
-            cwd = 'pytest'
+            cwd = os.path.join(REPO_DIR, 'pytest')
         else:
             print(f'Unrecognised test category ‘{fields[0]}’', file=sys.stderr)
             continue
         print("RUNNING COMMAND cwd=%s cmd = %s", (cwd, cmd))
-        subprocess.check_call(cmd, cwd=cwd)
+        subprocess.check_call(cmd, cwd=str(cwd))
```

Comment:
No need for `str()`. Pretty much everywhere where Python expects a path you can either pass `str` or `pathlib.Path` and the latter is preferred I’d say.

Reply:
Nice, I see that removing `str`, still works with `subprocess`